network/openvswitch_port.py

```diff
@@ -17,6 +19,11 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this software.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Portions copyright @ 2015 VMware, Inc.  All rights reserved.
```

Comment:
I asked this specific question to our lawyers. They say this is standard and does not conflict with GPL.
-Mark

From: Brian Coca 
Sent: Friday, July 10, 2015 3:37 PM
To: ansible/ansible-modules-extras 
Cc: Mark Hamilton 
Subject: Re: [ansible-modules-extras] Added support to assign attached mac address interface id and port options. (#702)

In network/openvswitch_port.py:

> @@ -17,6 +19,11 @@
>  #
>  # You should have received a copy of the GNU General Public License
>  # along with this software.  If not, see <http://www.gnu.org/licenses/>.
> +#
> +# Portions copyright @ 2015 VMware, Inc.  All rights reserved.
keep copyright with existing above, also not sure 'All rights reserved' works with GPL code

—
Reply to this email directly or view it on GitHub.